.codesandbox/ci.json

```diff
@@ -1,22 +1,67 @@
 {
   "buildCommand": "build",
   "packages": [
-    "/packages/paste-core/components/*",
-    "/packages/paste-core/layout/*",
-    "/packages/paste-core/primitives/*",
-    "/packages/paste-core/core-bundle",
-    "/packages/paste-design-tokens",
     "/packages/paste-icons",
+    "/packages/paste-core/core-bundle",
+    "/packages/paste-core/components/alert",
```

Comment:
Curious why you added the individual packages as well? Does it fix that "cant find X package" issue?

Reply:
Yup, essentially. What was happening before was it was building core and icons, but using all the packages from npm. This way codesandbox hosts all packages on it's package manager, and when we add a new package, we'll be able to test it in there too on the PR